### ISSUE TITLE
SelectCheckboxMenu: placeholder not supported with multiple=true

### ISSUE THREAD

**Issue by brunoaandrade** [primefaces/primefaces#3925]
## 1) Environment
PrimeFaces version: 6.2
Affected browsers: all

## 2) Expected behavior
A stylable hint should be displayed when label as value.

## 3) Actual behavior
The attribute "label" does not work with multiple="true"

**Comment by brunoaandrade** [comment]
The placeholder is shown when label attribute has value, but if multiple="true", nothing appears. I don't know if it is a new feature or a bug.
Either way, i updated the issue.

**Comment by tandraschko** [comment]
Please provide a bit more information and a example.
It's quite hard to follow you.

**Comment by chaseven** [comment]
I got the same issue here, definitely not that hard to reproduce...

`<p:selectCheckboxMenu value="#{bean.multiSelectedItems}" label="multiselect" multiple="false">`
will result in "multiselect" as label/placeholder
`<p:selectCheckboxMenu value="#{bean.multiSelectedItems}" label="multiselect" multiple="true">`
will result in a blank label/placeholder

**Comment by hberton** [comment]
up

**Comment by tandraschko** [comment]
@hberton feel free to implement it and provide a pull request.

**Comment by hberton** [comment]
@tandraschko 

Hello, follow my suggestion

add in "components.css"

```
.ui-selectcheckboxmenu-multiple-container:empty:after {
    content: attr(label);
}
```

SelectCheckboxMenuRenderer.java

```
protected void encodeMultipleLabel(FacesContext context, SelectCheckboxMenu menu, List<SelectItem> selectItems, boolean valid)
            throws IOException {
...
        writer.startElement("ul", null);
        // add line
	writer.writeAttribute("label", menu.getLabel(), null);
...
}
```
Example >
![image](https://user-images.githubusercontent.com/20052525/63274006-69975b80-c275-11e9-93c0-85bfb54d91a4.png)

**Comment by melloware** [comment]
Attached is a test case for proving the issue and the fix: 
[pf-3925.zip](https://github.com/primefaces/primefaces/files/3516819/pf-3925.zip)

**Comment by melloware** [comment]
I tested your fix in many scenarios including with and without multiple and with and without a label and it works great!  I tweaked your CSS slightly so it looks correct in the box.

**Comment by hberton** [comment]
@melloware 
thks!!!

I hope to contribute more in the future!
